lib/sentry.ex

```diff
@@ -97,40 +97,41 @@ defmodule Sentry do
   @max_hackney_connections Application.get_env(:sentry, :hackney_pool_max_connections, 50)
   @hackney_timeout Application.get_env(:sentry, :hackney_pool_timeout, 5000)
 
+  @type task :: {:ok, Task.t} | :error | :excluded | :ignored
+
   def start(_type, _opts) do
     children = [
       supervisor(Task.Supervisor, [[name: Sentry.TaskSupervisor]]),
       :hackney_pool.child_spec(Sentry.Client.hackney_pool_name(),  [timeout: @hackney_timeout, max_connections: @max_hackney_connections])
     ]
     opts = [strategy: :one_for_one, name: Sentry.Supervisor]
 
-
-    if @use_error_logger do
-      :error_logger.add_report_handler(Sentry.Logger)
-    end
+    if @use_error_logger, do: :error_logger.add_report_handler(Sentry.Logger)
```

Comment:
We've discussed this before, but please don't make cosmetic changes in specific PRs.